src/ui/components/seekbar/SeekBar.tsx

```diff
@@ -93,8 +108,11 @@ export const SeekBar = (props: SeekBarProps) => {
   };
 
   const onSlidingComplete = (value: number[]) => {
-    if (onScrubbing) onScrubbing(undefined);
-    setIsScrubbing(false);
+    if (onScrubbing) {
+      waitForSeeked(player).then( () => {
```

Comment:
Isn't this going to cause all kinds of weird edge cases if we hide the `seeked` event listener inside a non-hook?